plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -152,8 +152,12 @@ func (c *PodSecurityPolicyPlugin) Validate(a admission.Attributes) error {
 
 	pod := a.GetObject().(*api.Pod)
 
+	ValidatedPSP := ""
+	if pod.ObjectMeta.Annotations != nil {
```

Comment:
nil check isn't necessary, can just pass `pod.Annotations[psputil.ValidatedPSPAnnotation]` to computeSecurityContext

Reply:
Done.